pkg/formatters/detailsFormatter.go

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package formatters
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/galasa.dev/cli/pkg/galasaapi"
+)
+
+// -----------------------------------------------------
+// Detailed format.
+const (
+	DETAILS_FORMATTER_NAME = "details"
+)
+
+type DetailsFormatter struct {
+}
+
+func NewDetailsFormatter() RunsFormatter {
+	return new(DetailsFormatter)
+
+}
+
+func (*DetailsFormatter) GetName() string {
+	return DETAILS_FORMATTER_NAME
+}
+
+func (*DetailsFormatter) FormatRuns(runs []galasaapi.Run, apiServerUrl string) (string, error) {
+	var result string = ""
+	var err error = nil
+
+	if len(runs) < 1 {
+		return result, err
+	}
+
+	buff := strings.Builder{}
+
+	for i, run := range runs {
+		var duration string
+		var startTimeString string
+		var endTimeString string
```

Comment:
needs initialising to "" ? or is that implicit (I'm not a go master so don't know...) but if it's explicit then everyone knows for sure.

Reply:
i believe it is implicit but will change to explicit